orangecontrib/text/widgets/owpubmed.py

```diff
@@ -0,0 +1,410 @@
+import os
```

Comment:
Each widget module should have a 
```py
if __name__ == '__main__':
    ...  # Construct the app and show the widget with example data```
For example have a look at [any widget from orange](https://github.com/biolab/orange3/).